app/components/Dashboard/PriceHistoryPanel/index.js

```diff
@@ -1,34 +1,46 @@
 // @flow
 import { compose, withState } from 'recompose'
-import { withData, withActions } from 'spunky'
+import { withData, withActions, createBatchActions } from 'spunky'
 
 import PriceHistoryPanel from './PriceHistoryPanel'
 import priceHistoryActions from '../../../actions/priceHistoryActions'
+import pricesActions from '../../../actions/pricesActions'
 import withCurrencyData from '../../../hocs/withCurrencyData'
 import withProgressPanel from '../../../hocs/withProgressPanel'
+import withPricesData from '../../../hocs/withPricesData'
 import { ASSETS } from '../../../core/constants'
 
 type Duration = '1m' | '1w' | '1d'
 
-const mapPricesDataToProps = (prices, props) => ({
+const mapPriceHistoryDataToProps = (prices, props) => ({
   prices: prices[props.asset]
 })
 
+const mapPriceDataToProps = (staticPrices, props) => ({
+  staticPrice: staticPrices[props.asset]
+})
+
 const mapPriceHistoryActionsToProps = (actions, props) => ({
   setDuration: (duration: Duration) => {
     props.setDuration(duration)
     return actions.call({ duration, currency: props.currency })
   }
 })
 
+const batchActions = createBatchActions('priceHistoryPanel', {
```

Comment:
Could you move that into the `app/actions` directory?

Reply:
I'll do this to all the batchActions instances (as they all contain a TODO asking to move them into a separate action file) and submit it as another PR.